schemas/performance-artifact.json

```diff
@@ -82,9 +82,11 @@
                     "type": "array",
                     "title": "Extra options used in running suite",
                     "items": {
-                        "type": "string"
+                        "type": "string",
+                        "maxLength": 45
                     },
-                    "uniqueItems": true
+                    "uniqueItems": true,
+                    "maxItems": 18
```

Comment:
IMO we should lower this to something like 8. I don't see any good reason to allow so many extra options (generally treeherder errs on the side of being restrictive when it comes to what you can put in a data schema, which I think helps push people to reduce complexity overall -- a good thing).

Can you add a simple test or two for the edge cases here to `tests/etl/test_perf_schema.py`

Finally, we also need to mirror any changes to mozilla-central: https://searchfox.org/mozilla-central/source/testing/mozharness/external_tools/performance-artifact-schema.json. Generally in-tree tests check the perf data against that schema and then fail if they violate it.